### PR TITLE
clickhouse-cli: init at 0.3.6

### PR DESCRIPTION
**Pull request by johnpyp**
###### Motivation for this change

Add clickhouse-cli

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/database/clickhouse-cli/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  propagatedBuildInputs = with python3Packages; [
+    click
+    prompt_toolkit
+    pygments
+    requests
+    sqlparse
+  ];
+
+
```
Comment (skykanin) [inline pkgs/development/tools/database/clickhouse-cli/default.nix:22]: No need for this extra newline

**Review by skykanin — CHANGES_REQUESTED** [review]
Looks good to me except for a small nitpick. Package builds as expected on my machine.

**Review by skykanin — APPROVED** [review]
LGTM

File: pkgs/development/tools/database/clickhouse-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    requests
+    sqlparse
+  ];
+
+  checkPhase = ''
+    $out/bin/clickhouse-cli --help > /dev/null
+  '';
+
+  meta = with lib; {
+    homepage = https://github.com/hatarist/clickhouse-cli;
```
Comment (doronbehar) [inline pkgs/development/tools/database/clickhouse-cli/default.nix:27]: https://github.com/NixOS/rfcs/pull/45

Please quote the url.

**Comment by doronbehar** [comment]
@GrahamcOfBorg build clickhouse-cli

**Comment by Ma27** [comment]
Duplicate of #118862.

**Pull request closed without merging**